### PR TITLE
Changes for Code Review

### PR DESCRIPTION
**Pull request by snigdhagayatri**
Below are some of the changes I have made in your project
1. Inserted HTML code for table classes and tags to improve the look of the table
2. Handled code where the Store.Location is not being populated; added the Latitude and Longitude fields
3. Added .gitignore file which was missing in the project
4. Added comments to know the insights in the code
5. Enhanced the HTML code when "No Information Available" is being populated

Earlier Output

Output After changes

### CODE REVIEW

**Comment by snigdhagayatri** [comment]
Q: Summarize the three technical changes you made, and why.
A: Inserted HTML code for table classes and tags to improve the look of the table
Handled code where the Store.Location is not being populated; added the Latitude and Longitude fields
Added .gitignore file which was missing in the project
Added comments to know the insights in the code
Enhanced the HTML code when "No Information Available" is being populated

Q: List three items you learned from the code review.
A: 1. I learned thinking critically about code and its output
     2. I learned how to effectively use Git and GitHub
     3. I learned how to debug and see where the error was occurring or if the code has any null values. 

Q: Was the project available in GitHub on time?
A: Yes

Q: Does the project compile?
A: Yes

Q: Does the project have adequate documentation?
A: No

File: .gitignore.gitignore [hunk 1]
```diff
@@ -0,0 +1,356 @@
+
```
Comment (discospiff) [inline .gitignore.gitignore:1]: Good... gitignore is a good idea.

File: GroceryStores/GroceryStores/Pages/Index.cshtml [hunk 2]
```diff
@@ -8,32 +8,36 @@
 }
 
 <div class="text-center">
-    <h1 class="display-4">Welcome</h1>
+    <h1 class="display-4">Grocery Stores List</h1>
```
Comment (discospiff) [inline GroceryStores/GroceryStores/Pages/Index.cshtml:11]: This looks more descriptive.

File: GroceryStores/GroceryStores/Pages/Index.cshtml [hunk 3]
```diff
@@ -8,32 +8,36 @@
...
 
-    <table>
-        <tr>
+    <table class="table table-bordered table-striped">
+        <thead class="table-dark">
```
Comment (discospiff) [inline GroceryStores/GroceryStores/Pages/Index.cshtml:14]: Nice use of css themes/bootstrap.

File: GroceryStores/GroceryStores/Pages/Index.cshtml.cs [hunk 4]
```diff
@@ -8,6 +8,12 @@
 using BusinessOwnerDetails;
 using Microsoft.Extensions.Logging;
 
+/*Grocery Stores Project Combines two JSON files.
```
Comment (discospiff) [inline GroceryStores/GroceryStores/Pages/Index.cshtml.cs:11]: Use intellisense format, /// , not JavaDoc format, /* **/

File: GroceryStores/GroceryStores/Pages/SearchGroceryStores.cshtml [hunk 5]
```diff
@@ -67,29 +67,31 @@
...
+                                    <td>@grocery.Longitude</td>
+                                </tr>
                                     }
                                 }
                                 else
                                 {
                                     <tr class="danger">
                                         <td colspan="5" class="text-center">
-                                            No results found for term "@Model.Search"
+                                            <span class="font-weight-bolder text-danger">No results found for term "@Model.Search"</span>
```
Comment (discospiff) [inline GroceryStores/GroceryStores/Pages/SearchGroceryStores.cshtml:94]: Nice use of CSS styles, again.

**Comment by discospiff** [comment]
Several good changes here that enhance the program and decrease technical debt.  The team should consider merging these.

**Pull request merged**